docs/eventing/channels/channel-types-defaults.md

```diff
@@ -1,20 +1,35 @@
----
-title: "Channel types and defaults"
-weight: 60
-type: "docs"
-showlandingtoc: "true"
----
-
 # Channel types and defaults
+Knative uses two types of channels:
+
+
+* The generic Channel object
+* Channel implementations that each have their own custom resource definitions (CRDs), such as
+`InMemoryChannel` and `KafkaChannel`.
```

Comment:
If we're not doing `backticks` for Channel I'd leave them off for all other resources too and let's just consistently not use them for these, e.g. Service, Broker, KafkaChannel, PingSource, etc etc

Reply:
I think this was actually updated in the word list recently

Reply:
Okay, will do.